java/code/src/com/suse/manager/iss/v3/SyncController.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2024 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package com.suse.manager.iss.v3;
+
+import static com.suse.manager.webui.utils.SparkApplicationHelper.asJson;
+import static com.suse.manager.webui.utils.SparkApplicationHelper.json;
+import static com.suse.manager.webui.utils.SparkApplicationHelper.usingTokenAuthentication;
+import static spark.Spark.post;
+
+import com.suse.manager.webui.controllers.ECMAScriptDateAdapter;
+import com.suse.manager.webui.utils.gson.ResultJson;
+import com.suse.manager.webui.utils.token.Token;
+import com.suse.manager.webui.utils.token.TokenParsingException;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+
+import org.apache.http.HttpStatus;
+
+import java.util.Date;
+
+import spark.Request;
+import spark.Response;
+import spark.Spark;
+
+public class SyncController {
+
+    private static final Gson GSON = new GsonBuilder()
+        .registerTypeAdapter(Date.class, new ECMAScriptDateAdapter())
+        .serializeNulls()
+        .create();
+
+    /**
+     * Initialize the API routes
+     */
+    public void initRoutes() {
+        post("/iss/sync/ping", asJson(usingTokenAuthentication(this::ping)));
```

Comment:
In which context do you see this being used?

Reply:
Well, this was just to test if the token authentication was working. Indeed, this might get removed or improved when the API is finalized.

Reply:
got it. Can be seen as the example implementation at this stage, until we add other endpoints. I'm fine on keeping it, I was just wondering if it was needed for something :)